r/R/install-arrow.R

```diff
@@ -60,6 +62,7 @@ install_arrow <- function(nightly = FALSE,
                           minimal = Sys.getenv("LIBARROW_MINIMAL", FALSE),
                           verbose = Sys.getenv("ARROW_R_DEV", FALSE),
                           repos = getOption("repos"),
+                          prefer_runiverse = TRUE,
```

Comment:
I don't _love_ the option profusion here, but I do think going to r-universe first here is actually a nice plan. We don't have to muck with swapping to only source builds on macOS in order to get libarrow (and then also if someone has a borked gcc setup that would error too).

Reply:
If you take my other suggestion, you don't need to touch this function (though cleaning up the autobrew references is probably worthwhile anyway) or `arrow_repos()`.